### ISSUE TITLE
Questions about the loss function when it is a targeted attack.

### ISSUE THREAD

**Issue by WuGuommming** [Muzammal-Naseer/CDA#11]
Thanks for sharing your code :)

train.py line 146, the loss function of netG
`loss = criterion(model(normalize(adv)), targte_label) + criterion(model(normalize(img)), label)`
I think the second half of this loss function doesn't seem to be related to netG? This part should not influence the parameters of netG? I don't know if there is a problem with my understanding.

Another small problem is that when args.target is not equal to -1, target_label will be undefined in the code.
